atoma-proxy/src/server/streamer.rs

```diff
@@ -437,23 +431,21 @@ impl Stream for Streamer {
                                 "Error updating node latency performance: {}",
                                 e
                             );
-                            Error::new(format!("Error updating node latency performance: {}", e))
+                            Error::new(format!("Error updating node latency performance: {e:?}"))
```

Comment:
Do you need the `:?`?

Reply:
The Debug format often provides more detailed error information than Display